presto-native-execution/presto_cpp/presto_protocol/core/presto_protocol_core.h

```diff
@@ -2546,3 +2546,11 @@ enum class NodeState { ACTIVE, INACTIVE, SHUTTING_DOWN };
 extern void to_json(json& j, const NodeState& e);
 extern void from_json(const json& j, NodeState& e);
 } // namespace facebook::presto::protocol
+namespace facebook::presto::protocol {
```

Comment:
This class is code-generated. Please see the instructions at https://github.com/prestodb/presto/tree/master/presto-native-execution/presto_cpp/presto_protocol. You might have to create a special sub-directory and add this code there. Else another option is to making it a shared protocol class and adding to the yml file.

Reply:
@aditi-pandit since the response is defined in a plugin, where would you recommend putting the definition? I see now the protocol has been broken up into core and connectors. Does it make sense to add a new yml file, like "presto_sidecar.yml"?

Reply:
@BryanCutler : Adding a new yml file might be good.